test/test_sparse_csr.py

```diff
@@ -349,6 +349,11 @@ def test_coo_to_csr_convert(self, device, dtype, coalesced):
 
         self.assertEqual(coo_product, csr_product)
 
+        self.assertEqual(sparse_csr.crow_indices().shape[0], size[0] + 1)
+        self.assertEqual(sparse_csr.crow_indices().min(), 0)
+        self.assertEqual(sparse_csr.crow_indices().max(), sparse_csr.col_indices().shape[0])
+        self.assertTrue(all(sparse_csr.crow_indices()[1:] >= sparse_csr.crow_indices()[:-1]))
```

Comment:
These tests are testing CSR tensor invariants and hence are unnecessary because the invariants are tested when creating a CSR tensor, see
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/sparse/SparseCsrTensor.cpp#L23